core/language/en-GB/Help/render.tid

```diff
@@ -32,4 +32,4 @@ Notes:
 Examples:
 
 * `--render '[!is[system]]' '[encodeuricomponent[]addprefix[tiddlers/]addsuffix[.html]]'` -- renders all non-system tiddlers as files in the subdirectory "tiddlers" with URL-encoded titles and the extension HTML
-* `--render '.' 'tiddlers.json' 'text/plain' '$:/core/templates/exporters/JsonFile' 'exportFilter' '[tag[HelloThere]]'` -- renders the tiddlers tagged "HelloThere" to a JSON file named "tiddlers.json"
+* `--render '.' 'tiddlers.json' 'text/plain' '$:/core/templates/exporters/JsonFile' '' 'exportFilter' '[tag[HelloThere]]'` -- renders the tiddlers tagged "HelloThere" to a JSON file named "tiddlers.json"
```

Comment:
There is no missing quote. The format looks like this and it was OK -- So imo this PR can be closed with a "no fix"

```
--render <tiddler-filter> [<filename-filter>] [<render-type>] [<template>] [ [<name>] [<value>] ]*
```

Reply:
The reason is that if I don't add an empty quote, I will get an empty HTML. I saw that the documentation of --rendertiddler says this, so I also added an empty quote here, and the output HTML looks fine.

![image](https://github.com/user-attachments/assets/652bf31f-3aa6-4b06-99f1-f040c7e75475)


https://github.com/oeyoews/tiddlywiki-app/blob/efcd941227133b065994b72a8a4d216c90389f8c/src/utils/tiddlywiki.json#L17

Reply:
Actually this command line can't run on Windows, because of the $ sign. I just run it with a js program, and it won't run without empty quotes. I use the --rendertiddler screenshot to contrast with --render. I follow the current documentation and it doesn't work. Have you tried running this command?

Reply:
The syntax looks fine, but I’m not sure why there’s an issue when using `publishFilter`. I need to test it further.